package com.aptitude.exception;

public class TestAttemptNotFoundException extends RuntimeException {
    public TestAttemptNotFoundException(String message) {
        super(message);
    }
}
